BraveShared/Analytics/UserReferralProgram.swift

```diff
@@ -181,6 +190,20 @@ public class UserReferralProgram {
         return nil
     }
     
+    /// Passing string, attempts to derive a ref code from it.
+    /// Uses very strict matching.
+    /// Returns the sanitized code, or nil if no code was found
+    public class func sanitize(input: String?) -> String? {
+        guard var input = input, input.hasPrefix(self.clipboardPrefix) else { return nil }
+        
+        // +1 to strip off `:` that proceeds the defined prefix
+        input.removeFirst(self.clipboardPrefix.count + 1)
+        let valid = input.range(of: #"\b[A-Z]{3}[0-9]{3}\b"#, options: .regularExpression) != nil // true
```

Comment:
Leftover `// true` comment? 😄

Reply:
lol, yup, used for testing in a playground, thanks for pointing out.